plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -555,7 +555,9 @@ open class DefaultPageCreator(
                                                 kind = ContentKind.Parameters,
                                                 styles = mainStyles + ContentStyle.RowTitle
                                             )
-                                            comment(it.root)
+                                            if (it.root.children.isNotEmpty()) {
```

Comment:
I would create a helper like `TagWrapper.isEmpty() = children.isEmpty()` (Law of Demeter)